README.md

```diff
@@ -289,6 +289,10 @@ Specifing compression way for data of each record. Current accepted options are
 ### log_truncate_max_size
 Integer, default 1024. When emitting the log entry, the message will be truncated by this size to avoid infinite loop when the log is also sent to Kinesis. The value 0 means no truncation.
 
+### chomp_record
+Boolean. Default `false`. If it is enabled, formatter calls chomp and removes separator from the end of each record. This option is for compatible format with plugin v2.
+See [#142](https://github.com/awslabs/aws-fluent-plugin-kinesis/issues/142) for more details.
+
```

Comment:
Could you add more details for firehose?

Reply:
Thanks, updated like [this](https://github.com/awslabs/aws-fluent-plugin-kinesis/pull/172/files#diff-04c6e90faac2675aa89e2176d2eec7d8R292).